### DISCUSSION TITLE
Cómo en la azar mis apss para que trabajen automáticamente?.

### DISCUSSION THREAD

**Discussion by mauri-wuep** [community/community#43221] [Actions]
Cómo puedo enlazar con mis apss para que trabajen automáticamente?.

**Comment by LuisContrerasGlz** [comment]
Hola,

Para enlazar una aplicación con GitHub de manera que se ejecute de forma automática, necesitarás utilizar un servicio de integración continua (CI), puedes intentar lo siguiente:

1. Crea un repositorio en GitHub para tu aplicación y sube tu código a él.
2. Selecciona un servicio de integración continua, como Travis CI o CircleCI.
3. Sigue las instrucciones del servicio de integración continua para configurar un archivo de configuración en tu repositorio de GitHub. 4. Este archivo le indicará al servicio cómo compilar y probar tu código.
5. Haz un commit y un push de tu archivo de configuración a tu repositorio de GitHub.
6. Una vez que el servicio de integración continua haya recibido la actualización del repositorio, comenzará a compilar y probar automáticamente tu código cada vez que se realicen cambios en el repositorio.

Espero te sirva mi respuesta :-)
